pandas/io/tests/test_sql.py

```diff
@@ -2477,6 +2477,25 @@ def test_write_row_by_row(self):
         result.index = frame.index
         tm.assert_frame_equal(result, frame)
 
+    def test_chunksize_read_type(self):
+        _skip_if_no_pymysql()
+        frame = tm.makeTimeDataFrame()
+        frame.index.name = "index"
+        drop_sql = "DROP TABLE IF EXISTS test"
+        cur = self.conn.cursor()
+        cur.execute(drop_sql)
+        sql.to_sql(frame, name='test', con=self.conn, flavor='mysql')
+        query = "select * from test"
+        chunksize = 5
+        chunk_gen = pd.read_sql_query(sql=query, con=self.conn,
+                                      chunksize=chunksize, index_col="index")
+        chunk_df = pd.DataFrame()
+        try:
+            chunk_df = next(chunk_gen)
+        except TypeError:
+            assert False, "expected list of tuples, got tuple of tuples"
```

Comment:
I don't think the try .. except is necessary, the test will just error if this does not work, which is OK as well